af/fastpath/fastpath/core.py

```diff
@@ -49,6 +49,7 @@
 import fastpath.portable_queue as queue
 
 import fastpath.utils
+from fastpath.utils import dget_or as g_or
```

Comment:
why?

Reply:
It's a simple getter. The code is moving toward lenient getters that can handle null values or missing keys - and adding unit tests. At the end of the migration we can do some refactoring.

Reply:
Right, but why are you aliasing it to `g_or` instead of using the `dget_or` name as is done in other parts of the code?

Reply:
It was written like that earlier on elsewhere and I wanted to avoid mixing too much feature development and code cleanup in the same PR.